active/0046-improved-release-process.md

```diff
@@ -0,0 +1,55 @@
+# Improved Release Process
+
+- Start Date: 2016-03-26
+- RFC PR: #46
+ 
+## Summary & Motivation
+
+ember-cli has followed an ad hoc release process throughout its existence which has made it difficult to know exactly when releases would come out, what features would and would not be supported, and the degree to which it would support existing Ember applications. With the proposal for lockstep SemVer there were ideals of guaranteeing compatibility, which we have mostly met, but that resulted in making decisions of delaying an official 2.X release of ember-cli to avoid additional major version bumps.
+
+We propose that we adopt a pattern similar to Ember itself in order to align with the expectations of the Ember community, more-clearly communicate around our release lifecycle, and provide rigor around our support structure. Anything which is not specifically called out as a difference in this document is inferred to be following the patterns specified by Ember itself.
+
+## Channel Design
+
+To begin there will be three separate channels: canary, beta, and release. We intend to investigate an LTS channel after this process has matured.
+
+- **Canary**: represents the latest work in ember-cli, and is synonymous with the `HEAD` of the `master` branch and is the least stable of all channels.
```

Comment:
I know @rwjblue would like to rename master to `canary` and make it the default branch. I personally could go either way, I'm curious what the community wants.